src/commands/lambda/flare.ts

```diff
@@ -1,14 +1,59 @@
 import {Command} from 'clipanion'
 
-import {renderLambdaFlareHeader} from './renderers/flare-renderer'
+import {renderError, renderLambdaFlareHeader} from './renderers/flare-renderer'
 
 export class LambdaFlareCommand extends Command {
   private isDryRun = false
+  private isInteractive = false
+  private allFunctions = false
+  private functions: string[] = []
+  private region = ''
+  private apiKey = ''
+  private caseID = ''
+  private email = ''
 
+  /**
+   * @returns 0 if the command ran successfully, 1 otherwise.
+   */
   public async execute() {
     this.context.stdout.write(renderLambdaFlareHeader(this.isDryRun))
+
+    if (!this.validateCommand()) {
+      return 1
+    }
+
+    return 0
+  }
+
+  /**
+   * @returns true if the command is valid, false otherwise.
+   * Prints an error message if the command is invalid.
+   */
+  private validateCommand = () => {
+    if (this.isInteractive) {
+      return true
+    } else if (this.functions.length === 0 && !this.allFunctions) {
+      this.context.stdout.write(renderError('No functions specified. [-f,--function] or [--allFunctions]'))
+    } else if (this.region === '') {
+      this.context.stdout.write(renderError('No region specified. [-r,--region]'))
+    } else if (this.apiKey === '') {
+      this.context.stdout.write(renderError('No API key specified. [--api-key]'))
+    } else if (this.email === '') {
+      this.context.stdout.write(renderError('No email specified. [-e,--email]'))
+    } else {
+      return true
+    }
+
+    return false
   }
 }
 
 LambdaFlareCommand.addPath('lambda', 'flare')
 LambdaFlareCommand.addOption('isDryRun', Command.Boolean('-d,--dry'))
+LambdaFlareCommand.addOption('isInteractive', Command.Boolean('-i,--interactive'))
+LambdaFlareCommand.addOption('allFunctions', Command.Boolean('--allFunctions'))
```

Comment:
Standard for CLI options is kebab case.
Also let's remove this option until an implementation to flare all functions is available.